src/client/src/api/fetchApiV2.js

```diff
@@ -85,6 +85,15 @@ export const fetchLithologicalDescriptionsByProfileId = async profileId => {
   );
 };
 
+//casings
```

Comment:
PP: Der Kommentar sagt nicht viel aus und ich würde ihn daher löschen.

Reply:
![image](https://user-images.githubusercontent.com/49144408/234783657-626a3ff8-991c-411e-aaa1-b1b8ed880ed4.png)
 Wir haben das ganze file mit den titeln etwas strukturiert, ich lasse es also drin. Aber es wäre eine Idee das langsam mal in einzelne files zu unterteilen. Werde ich im nächsten Schritt angehen.

Reply:
Alles klar. Dann braucht es lediglich noch einen Whitespace zwischen `//` und dem Kommentar → `// casings`